src/Language/PureScript/Docs/Render.hs

```diff
@@ -70,12 +70,12 @@ renderDeclarationWithOptions opts Declaration{..} =
   renderType' = renderTypeWithOptions opts
   renderAlias (P.Qualified mn alias)
     | mn == currentModule opts =
-        P.foldFixityAlias P.runIdent P.runProperName P.runProperName alias
+        P.foldFixityAlias P.runIdent P.runProperName (("type " ++) . P.runProperName) alias
```

Comment:
Yeah there used to be an `either` here before I introduced a real sum type, having a fold seemed slightly more elegant than explicit pattern matching in this case. I just noticed when working on the other stuff that it was mis-rendering type operator aliases so fixed that while I was at it.